client/stylesheets/abstracts/_variables.scss

```diff
@@ -13,6 +13,11 @@ $gap-small: 12px;
 $gap-smaller: 8px;
 $gap-smallest: 4px;
 
+// Header
+$small-header-height: 50px;
```

Comment:
Awesome- thanks for adding these 😄 

While we're here, can we also use these for the activity panel and breadcrumb heights that are set manually?

Reply:
> While we're here, can we also use these for the activity panel and breadcrumb heights that are set manually?

Sure thing! I will do that in a follow-up if that sounds ok, so we don't block this PR.